### ISSUE TITLE
New language: Lithuanian

### ISSUE THREAD

**Issue by sinia** [squidfunk/mkdocs-material#4202]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translation

```jinja
{% macro t(key) %}{{ {
  "language": "lt",
  "direction": "ltr",
  "clipboard.copy": "Kopijuoti į iškarpinę",
  "clipboard.copied": "Nukopijuota į iškarpinę",
  "edit.link.title": "Redaguoti šį puslapį",
  "footer.previous": "Ankstesnis",
  "footer.next": "Sekantis",
  "footer.title": "Poraštė",
  "header.title": "Antraštė",
  "meta.comments": "Komentarai",
  "meta.source": "Išeitinis kodas",
  "nav.title": "Navigacija",
  "search.placeholder": "Paieška",
  "search.share": "Dalintis",
  "search.reset": "Išvalyti",
  "search.result.initializer": "Paieškos inicijavimas",
  "search.result.placeholder": "Įveskite norėdami pradėti paiešką",
  "search.result.none": "Atitinkančių dokumentų nerasta",
  "search.result.one": "1 atitinkantis dokumentas",
  "search.result.other": "# atitinkantys dokumentai",
  "search.result.more.one": "Dar 1 šiame puslapyje",
  "search.result.more.other": "Dar # šiame puslapyje",
  "search.result.term.missing": "Nerasta",
  "search.title": "Paieška",
  "select.language.title": "Pasirinkti kalbą",
  "select.version.title": "Pasrinkti versiją",
  "skip.link.title": "Pereiti prie turinio",
  "source.link.title": "Eiti į saugyklą",
  "source.file.date.updated": "Paskutinis atnaujinimas",
  "source.file.date.created": "Sukurta",
  "tabs.title": "Skirtukai",
  "toc.title": "Turinys",
  "top.title": "Grįžti į viršų"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks! Added in eef9bdece.

**Comment by squidfunk** [comment]
Released as part of 8.4.0.
